build/webpack/defaultWebpackConfig.js

```diff
@@ -113,7 +113,7 @@ module.exports = class WebpackDefaults {
 				path: path.resolve(__dirname, '../../dist/')
 			},
 			resolve: {
-				extensions: ['.tsx', '.ts', '.js', '.jsx', '.json', 'scss', 'html'],
+				extensions: ['.js', '.jsx', '.tsx', '.ts', '.json', 'scss', 'html'],
```

Comment:
This needs to be in the correct order for the build else it creates files with duplicate extensions - myfile.ts.js